syntax = "proto2";
package android.trusty.keymaster.fuzz;

import "fuzz/keymaster_defs.proto";
import "fuzz/requests.proto";

message Test {
    oneof test {
        SigningTest signing_test = 1;
    }
}

message SigningTest {
    required BeginOperationRequest begin_op = 1;
    required UpdateOperationRequest update_op = 2;
    required FinishOperationRequest finish_op = 3;
}
